qualtran/surface_code/ccz2t_cost_model.py

```diff
@@ -175,35 +174,41 @@ def get_ccz2t_costs(
     Note that this function can return failure probabilities larger than 1.
 
     Args:
-        n_magic: The number of magic states (T, Toffoli) required to execute the algorithm
+        n_logical_gates: The number of algorithm logical gates.
         n_algo_qubits: Number of algorithm logical qubits.
         phys_err: The physical error rate of the device.
         cycle_time_us: The number of microseconds it takes to execute a surface code cycle.
         factory: magic state factory configuration. Used to evaluate distillation error and cost.
         data_block: data block configuration. Used to evaluate data error and footprint.
     """
-    err_model = qec.LogicalErrorModel(
-        qec_scheme=qec.FowlerSuperconductingQubits, physical_error=phys_err
+    from qualtran.surface_code import FowlerSuperconductingQubits, LogicalErrorModel
+
+    err_model = LogicalErrorModel(qec_scheme=FowlerSuperconductingQubits, physical_error=phys_err)
```

Comment:
so we can't use CCZ2T with another QEC?

Reply:
This function is kept for backwards compatibility and is anchored to the ccz2t paper. Stay tuned!